src/platform/packages/shared/response-ops/rule_params/common/data_view_spec_schema/data_view_spec_schema.ts

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+import { schema } from '@kbn/config-schema';
+import { MAX_DATA_VIEW_FIELD_DESCRIPTION_LENGTH } from './constants';
+import { fieldSpecSchema, runtimeFieldSchema, serializedFieldFormatSchema } from './schemas';
```

Comment:
super nit: if these schemas are going to be used only in `dataViewSpecSchema `. They could be defined here instead of in a separate file.